hnn_core/network_builder.py

```diff
@@ -324,6 +337,13 @@ def _build(self):
         self.move_cells_to_pos()  # position cells in 2D grid
         self._connect_celltypes()
 
+        if self._expose_imem:
+            from .lfp import _TransmembraneCurrentHandler
+            self.imem_handler = _TransmembraneCurrentHandler(pc=_PC,
+                                                             cvode=_CVODE)
```

Comment:
This really boosts the speed. I also think it makes semantic sense for the transmembrane currents to be "handled" on their own. What those currents are used for may vary, but here obviously we need them for LFP.

@jasmainak I'm pushing updated versions of our example scripts that use this new API. Compare speed before and after ;)